src/server/metrics/index.js

```diff
@@ -0,0 +1,17 @@
+import updateGitHubUsersTotal from './github-users';
+
+let existingInterval = null;
+
+export default async function startAllMetrics() {
+  if (existingInterval !== null) {
+    clearInterval(existingInterval);
+  }
+
+  async function updateAllMetrics() {
```

Comment:
Nit picking a bit, but can't `updateAllMetrics` be defined outside of `startAllMetrics` scope? It doesn't seem to rely on any internal scope, so it doesn't need to be redefined every time `startAllMetrics` is called.

Of course, we call `startAllMetrics` just once, so it shouldn't really be an issue.